libnavui-dropin/src/main/res/layout/mapbox_navigation_view_layout.xml

```diff
@@ -99,7 +99,7 @@
                     android:layout_width="@dimen/mapbox_actionList_width"
                     android:layout_height="wrap_content"
                     android:gravity="top|right"
-                    android:layout_marginTop="8dp"
+                    android:layout_marginTop="5dp"
```

Comment:
Oh, you changed everything. That will probably require some changes in my PR as well. Should we move it to dimens?